docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc

```diff
@@ -114,6 +115,29 @@ logs informational messages, warnings, errors, and critical errors.
 
 . Click **Apply changes** to apply the updated logging level to the agent.
 
+[discrete]
+[[collect-agent-diagnostics]]
+= Collect {agent} diagnostics
+
+{fleet} provides the ability to remotely generate and gather an {agent}'s diagnostics bundle.
+An agent can gather and upload diagnostics if it is online in a `Healthy` or `Unhealthy` state.
+Diagnostics bundles can be downloaded from {fleet} to be viewed locally.
+
+. In {kib}, go to **Management -> {fleet} -> Agents**.
+
+. In the **Host** column, click the agent's name.
+
+. Click the **Diagnostics** tab.
+
+. Click the `Request diagnostics .zip` button.
++
+[role="screenshot"]
+image::images/collect-agent-diagnostics.png[Collect agent diagnostics under agent details]
+
+Any in-progress or previously collected bundles for the {agent} will be listed below.
```

Comment:
```suggestion
Any in-progress or previously collected bundles for the {agent} will be listed on this page.
```

Reply:
Generally a best practice to avoid using above, below, etc to describe areas of the UI.